src/styles/tab.css

```
@@ -30,3 +30,21 @@
+.result-none {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 200px;
+  font-size: 18px;
+  gap: 25px;
+}
+
+.result-none-text {
+  font-family: "Poppins";
```

Stylelint (17.5.0)

[error] Unexpected quotes around "Poppins" (font-family-name-quotes)

(font-family-name-quotes)